doc/source/manual/xtrabackup_sst.rst

```diff
@@ -35,6 +35,13 @@ under ``[sst]``.
      on either donor or joiner node, not during ``mysqld`` startup.
 
    * SST options must be specified in the main :file:`my.cnf` file.
+   
+The following options can be added to the [sst] section of the configuration file to control SST timeouts:
+
+* ``sockopt``
+* ``donor-timeout`` - the value of the `connect-timeout` on the donor node. The default valueis 10 seconds.
```

Comment:
In simple: Timeout for the donor to connect to joiner. The default value is 10 seconds.